_faq/security/auth-mix-and-match.md

```diff
@@ -2,7 +2,26 @@
 title: Can I enable Istio Auth with some services while disable others in the same cluster?
 weight: 30
 ---
-Starting with release 0.3, you can use service-level annotations to disable (or enable) Istio Auth for particular service-port.
+{% include home.html %}
+
+Starting with Istio 0.8, you can use [authentication policy]({{home}}/docs/concepts/security/authn-policy.html) to enable (or disable) mTLS per service. For example, the policy below will disable mTLS on port 9080 for service `details`
+
+```command
```

Comment:
command -> bash
Remove the $

Can't use 'command' for this case.

Reply:
Done